### PR TITLE
Added quaternion class for later use

### PR DESCRIPTION
**Pull request by pgemuende**
The class may be helpful for other users, that's why I decided to push it already.

### CODE REVIEW

File: src/Quaternion.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/*
+  Copyright (C) 2012,2013
```
Comment (junghans) [inline src/Quaternion.cpp]: 2015 missing!

File: src/Quaternion.hpp [hunk 2]
```diff
@@ -0,0 +1,291 @@
+/*
+  Copyright (C) 2012,2013
```
Comment (junghans) [inline src/Quaternion.hpp]: 2015 missing!

File: src/Quaternion.hpp [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+  inline real Quaternion::getItem(int i) const
+  { return this->at(i); }
+  // { if (i==0) {
+  //     return real_part; 
+  //   } else if (i>0 && i<4) {
+  //     return unreal_part[i-1];
+  //   } else {
+  //     throw std::out_of_range("Quaternion::getItem");
+  //   }
+  // }
```
Comment (junghans) [inline src/Quaternion.hpp]: Remove that block!

File: src/Quaternion.py [hunk 4]
```diff
@@ -0,0 +1,253 @@
+#  Copyright (C) 2012,2013
```
Comment (junghans) [inline src/Quaternion.py]: 2015 missing!

File: src/Quaternion.py [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+#      Max Planck Institute for Polymer Research
+#  Copyright (C) 2008,2009,2010,2011
```
Comment (junghans) [inline src/Quaternion.py]: 2015 missing!

**Comment by junghans** [comment]
Please fix these things in the next pull request!

**Comment by pgemuende** [comment]
I'll do that, thanks for reviewing!

**Pull request merged**